config/license.js

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2020 Adobe. All rights reserved.
+Copyright <%= YEAR %> Adobe. All rights reserved.
```

Comment:
This matches both the eslint syntax and works for the stylelint plugin we've written